src/tools/dev/scripts/visit-install-open

```diff
@@ -327,7 +327,6 @@ if [ "$poodle" = "true" ]
 then
    if [ "$test" = "no" ]
    then
-      cp /usr/tmp/$user/poodle/visitbuild/visit$ver2.linux-x86_64.tar.gz visit$ver2.linux-x86_64-poodle.tar.gz
```

Comment:
where will you get the tarball then?

Reply:
current dir -- but we could modify to copy from the /usr/workspace/visit/visit loc in the future

Reply:
huh - I had to copy from the result dir we made in /usr/workspace

does visit-build-open put it in the current dir?

Reply:
fixed